tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -67,7 +67,7 @@ type Message = {
 type Parameters = {
     compat_mode : [2],
     psk_mode : Bit,
-    retry : Bit,
+    retry : [8],
```

Comment:
Why does this need to be a byte instead of a bit? I remember you mentioned a problem with SAW.

Reply:
Joey mentioned it has something to do with how LLVM deals with Bits. I'm not entirely clear on the reason though. We can ask Galois to find out if this is being fixed in a new release.

Reply:
that doesn't quite make sense to me as the others are still bits and seem to be working?

Reply:
I'm not sure we're using the other ones in the SAW proof.

In this case it's because the way LLVM translates bitfields can be a bit challenging to reason about. The compiler often translates a byte of bitfields as an i8 and then uses masking to deal with the bits individually. In this case things seem to work out ok because of that translation, but that's not always a general solution.

We'll double check this specific case and get back to you

Reply:
@jldodds is correct, SAW has this limitation when handling bitfields, and none of the other flags are used in the SAW proof yet. As far as I can tell, the others are not bitfields. For example, `no_client_cert` is a specific bit in `handshake_type`.

In this particular case, I would suggest to keep it as `Bit` here, and set it below to `conn.handshake.hello_retry_request != 0`.